src/Components/PageLayout/PageLayout.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
```

Comment:
Designs look good

The only thing that needs to be changed is the email/password title locations

![image](https://user-images.githubusercontent.com/30921670/72703319-48d47a00-3b23-11ea-95ae-243fbf141d96.png)

Reply:
Ok. The design have no label so I wasn't sure what would happen if the fields are populated. Maybe have this label but position it a bit up?